sermant-plugins/sermant-mq-grayscale/sermant-mq-grayscale-plugin/src/main/resources/META-INF/services/io.sermant.core.plugin.agent.declarer.PluginDeclarer

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (C) 2024-2024 Sermant Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+io.sermant.mq.grayscale.declarer.RocketMqProducerGrayMessageHookDeclarer
+io.sermant.mq.grayscale.declarer.PullConsumerSubscriptionUpdateDeclarer
+io.sermant.mq.grayscale.declarer.MqSchedulerRebuildSubscriptionDeclarer
+io.sermant.mq.grayscale.declarer.MqLitePullConsumerSubscribeDeclarer
+io.sermant.mq.grayscale.declarer.MqPullConsumerFetchDeclarer
+io.sermant.mq.grayscale.declarer.MqPushConsumerSubscribeFetchDeclarer
```

Comment:
your Declarers are not committed,  so do not contains this

Reply:
fixed